lib/commands/create.js

```diff
@@ -128,6 +113,23 @@ async function main (fabricator, debugOut, argv) {
       tokens,
       specification
     })
+
+    // make sure the cluster has a PX00 mainenance window channel, and the
+    // maintenance startTime is correct
+    const clusterClient = new container.v1.ClusterManagerClient({
```

Comment:
nit: `clusterClient` threw me for a loop because i missed this line and assumed it was an instance of cluster-info-client and was really confused as to how the maintenance window was actually being set